### PR TITLE
feat(environments): Filter stats overview chart by environment

### PR DESCRIPTION
**Pull request by lynnagara**
Depends on https://github.com/getsentry/sentry/pull/7352

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectDashboard.jsx [hunk 1]
```diff
@@ -152,7 +152,11 @@ const ProjectDashboard = createReactClass({
           </div>
           <h3>{t('Overview')}</h3>
         </div>
-        <ProjectChart dateSince={dateSince} resolution={resolution} />
+        <ProjectChart
+          dateSince={dateSince}
+          resolution={resolution}
+          environment={this.props.environment}
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/projectDashboard.jsx]: Am I missing something, or is `environment` not a declared prop for `ProjectDashboard`? (It is accessed via `context` on [L98](https://github.com/getsentry/sentry/pull/7354/files#diff-abb03cf8790950f418bd32d97abc5a91L98) though.)

**Review by benvinegar — CHANGES_REQUESTED** [review]
See comment.

File: src/sentry/static/sentry/app/views/projectDashboard.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectDashboard.jsx]: This depends on the changes in https://github.com/getsentry/sentry/pull/7352

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectDashboard.jsx]: @benvinegar Rebased, please take a look

**Review by benvinegar — APPROVED** [review]

**Pull request merged**